en/sql/jsp.rst

```diff
@@ -306,7 +343,157 @@ or
 
     cubrid.jdbc.driver.CUBRIDDriver.getDefaultConnection();
 
-If you connect to the database using the JDBC driver as shown above, the transaction in the Java stored function/procedure is ignored. That is, database operations executed in the Java stored function/procedure belong to the transaction that called the Java stored function/procedure. In the following example, **conn.commit()** method of the **Athlete** class is ignored.
+.. note::
+
+    The server-side JDBC is already registered, you don't need to call "Class.forName("cubrid.jdbc.driver.CUBRIDDriver")"
```

Comment:
The following is different from ko.
Double quotes are surrounded by duplicates.

```
ko: Class.forName("cubrid.jdbc.driver.CUBRIDDriver")
en: "Class.forName("cubrid.jdbc.driver.CUBRIDDriver")"
```

Reply:
This is to let them know that it is a code...